tutorials/cyber/raw_data_generator/run_raw_data_generator.ini

```diff
@@ -0,0 +1,4 @@
+[HDFS]
```

Comment:
same question as above, do we expect this notebook to try connecting to an hdfs instance?

Reply:
Raw data generator is not integrated into the notebook. It just generates raw data from the LANL data source files and expects to read and write to HDFS. In our dev environment we have saved LANL data source files in HDFS. I can remove hdfs requirement and have it write to local file system if thats easier?

Reply:
I have updated the script to write to the local file system